src/components/internal/Card/Card.tsx

```diff
@@ -39,24 +76,34 @@ const Card = ({
                 classNameModifiers
             )}
             data-testid={testId}
+            {...cardContainerAttributes}
         >
             {(title || renderHeader) && (
-                <Tag className={CARD_HEADER}>
+                <Tag className={classNames(CARD_HEADER, { [`${CARD_HEADER}--expandable`]: expandable })}>
                     {(title || renderHeader) && (
-                        <div className={CARD_HEADER_CONTENT}>
+                        <div className={classNames(CARD_HEADER_CONTENT, { [`${CARD_HEADER_CONTENT}--expandable`]: expandable })}>
+                            {expandable && (
+                                <div className={CARD_TOGGLE_CLASS}>
+                                    {showContent ? <ChevronUp role="presentation" /> : <ChevronDown role="presentation" />}
+                                </div>
+                            )}
                             {renderHeader ? renderHeader : <span className={CARD_TITLE}>{title}</span>}
                             {subTitle && <div className={CARD_SUBTITLE}>{subTitle}</div>}
                         </div>
                     )}
                 </Tag>
             )}
-            <div
-                className={classNames(CARD_BODY, {
-                    [CARD_BODY_WITH_TITLE]: title || renderHeader,
-                })}
-            >
-                {children}
-            </div>
+            {(!expandable || (expandable && showContent)) && (
```

Comment:
Consider simplifying this check as follows:
```suggestion
            {(!expandable || showContent) && (
```